commit ce517df7029a9e7c8168a1883da7c346ef61aa57
Merge: 43c298a 2242a84
Author: alexborro <alexborro@gmail.com>
Date:   Tue Jul 1 13:21:53 2014 -0300

    Merge pull request #989 from bq/cancel_heatup
    
    Changed the declaration of the variable "cancel_heatup" to Marlin_main.cpp to fix issue on commit #965

diff --git a/Marlin/Marlin_main.cpp b/Marlin/Marlin_main.cpp
index 3e44168..0367eb4 100644
--- a/Marlin/Marlin_main.cpp
+++ b/Marlin/Marlin_main.cpp
@@ -276,6 +276,8 @@ int EtoPPressure=0;
   float delta_segments_per_second= DELTA_SEGMENTS_PER_SECOND;
 #endif					
 
+bool cancel_heatup = false ;
+
 //===========================================================================
 //=============================Private Variables=============================
 //===========================================================================
diff --git a/Marlin/ultralcd.cpp b/Marlin/ultralcd.cpp
index bb7dd0f..18c8588 100644
--- a/Marlin/ultralcd.cpp
+++ b/Marlin/ultralcd.cpp
@@ -19,8 +19,6 @@ int absPreheatHotendTemp;
 int absPreheatHPBTemp;
 int absPreheatFanSpeed;
 
-bool cancel_heatup = false ;
-
 #ifdef ULTIPANEL
 static float manual_feedrate[] = MANUAL_FEEDRATE;
 #endif // ULTIPANEL
